### PR TITLE
bugfix/#2712 Renaming UBS nav link

### PR DESCRIPTION
**Pull request by paslavskyi9roman**
Navigation-link has been renamed: UBS -> UBS courier
зображення

original task: https://github.com/ita-social-projects/GreenCity/issues/2712

### CODE REVIEW

File: src/app/component/layout/components/header/header.component.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
               </a>
             </li>
             <li role="link">
-              <a [routerLink]="['/ubs']" routerLinkActive="active-link" routerLinkActiveOptions="{exact:true}"> UBS </a>
+              <a [routerLink]="['/ubs']" routerLinkActive="active-link" routerLinkActiveOptions="{exact:true}"> UBS courier</a>
             </li>
```
Comment (yuriiLisunov) [inline src/app/component/layout/components/header/header.component.html:43]: i think, we must to change this magic string by interpolation like in other routes ( e.g.: {{ 'user.lower-nav-bar.my-habits' | translate }})

Comment (paslavskyi9roman) [reply src/app/component/layout/components/header/header.component.html:43]: As far as I know, we don't have this name in our translation file yet. so this is just a plug to display the appropriate name for a link at this time

Comment (yuriiLisunov) [reply src/app/component/layout/components/header/header.component.html:43]: can you change translation file and add this name?

Comment (paslavskyi9roman) [reply src/app/component/layout/components/header/header.component.html]: Translation has been added with this commit 49872ee153b5ee428da2c588b04d0b4afe73fdb2

**Comment by JimHolman1199** [comment]
please change branch name and PR title to have number of issue

File: src/assets/i18n/en.json [hunk 2]
```diff
@@ -253,6 +253,7 @@
     "add-cafe": "Add a cafe",
     "admin": "Administrator",
     "my-habits": "My habits",
+    "ubs": "UBS courier",
```
Comment (JimHolman1199) [inline src/assets/i18n/en.json:256]: we have the same text in 4 places in 1 file, how about having only 1 ?

Comment (paslavskyi9roman) [reply src/assets/i18n/en.json]: unnecessary translations were removed with this commit 17b2ed1a78d91e4e158a46199ff72ff2f89963c1

**Review by JimHolman1199 — CHANGES_REQUESTED** [review]
please change branch name and PR title to have number of issue

**Pull request closed without merging**